rotkehlchen/assets/utils.py

```diff
@@ -106,7 +106,6 @@ def get_or_create_evm_token(
 def get_asset_by_symbol(
         symbol: str,
         asset_type: Optional[AssetType] = None,
-        preferred_chain: ChainID = ChainID.ETHEREUM,
```

Comment:
was not used anywhere?

Reply:
no, it was not being used

Reply:
Shuldn't it always be given as an argument?. Cause now you still keep it optional. But I see you are actually passing it everywhere now in this PR

Reply:
No. Before it was default eth and was breaking the logic in some places. Now is only passed in the places where we need to use the symbol but we know the chain. I added it only to two places and by default is None. Is in places where you know the chain where the asset is when I use the argument

Reply:
all right